src/1/example-bot.md

```diff
@@ -43,14 +43,13 @@ cts.Cancel();
 async Task HandleUpdateAsync(ITelegramBotClient botClient, Update update, CancellationToken cancellationToken)
 {
     // Only process Message updates: https://core.telegram.org/bots/api#message
-    if (update.Type != UpdateType.Message)
+    if (!(update.Message is { } message))
         return;
     // Only process text messages
-    if (update.Message!.Type != MessageType.Text)
+    if (!(message.Text is { } messageText))
```

Comment:
I think you can use `is not` operator here which is easier to read than operator `!`.